<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="SLB-2TZTXT2" failures="2" tests="2" name="Test(failed)" time="0.002" errors="0" timestamp="2021-04-29T19:39:37 IST">
  <testcase classname="TestNGDemo.LoginTest" name="ValidateHomePage" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGDemo.LoginTest.ValidateHomePage(LoginTest.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at TestNGDemo.RunFailedTestcases.runfailedtestcases(RunFailedTestcases.java:24)
... Removed 52 stack frames]]>
    </failure>
  </testcase> <!-- ValidateHomePage -->
  <testcase classname="TestNGDemo.MakePaymentTest" name="paymentSuccessful" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGDemo.MakePaymentTest.paymentSuccessful(MakePaymentTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at TestNGDemo.RunFailedTestcases.runfailedtestcases(RunFailedTestcases.java:24)
... Removed 52 stack frames]]>
    </failure>
  </testcase> <!-- paymentSuccessful -->
  <testcase classname="TestNGDemo.LoginTest" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
  <testcase classname="TestNGDemo.MakePaymentTest" name="makePayment" time="0.0">
    <ignored/>
  </testcase> <!-- makePayment -->
</testsuite> <!-- Test(failed) -->
